.github/workflows/rolling-compatibility-humble-binary-build.yml

```
@@ -46,5 +46,7 @@ jobs:
     with:
       ros_distro: humble
       ros_repo: testing
-      upstream_workspace: ros2_controllers.rolling.repos
+      upstream_workspace: |
+        ros2_controllers-not-released.humble.repos
+        ros2_controllers.rolling.repos
```

```suggestion
      upstream_workspace: ros2_controllers.rolling.repos
```
This never got released, and somehow is needed on humble but not on later distros
https://github.com/ros/filters/commit/34f154b89aac40c74a1dcf0166338f4837b4590e

---

There was a release 2 days ago. Let's see what happens